lib/jquery.jsonp-2.4.min.d.ts

```diff
@@ -0,0 +1,40 @@
+// Available options and types derived from https://github.com/jaubourg/jquery-jsonp/blob/master/doc/API.md
```

Comment:
I could not find official types from this plugin so I rolled my own.  Happy to replace with others if some can point them out.